projects/packages/jitm/src/class-pre-connection-jitm.php

```diff
@@ -53,7 +53,7 @@ private function filter_messages( $message_path ) {
 				'message'     => $message['message'],
 				'description' => $message['description'],
 				'list'        => array(),
-				'icon'        => 'jetpack',
+				'icon'        => isset( $message['icon'] ) ? $message['icon'] : 'jetpack',
```

Comment:
Should we check whether the icon is one of an array of accepted values, instead of passing anything that may be provided?

Reply:
The `JITM::generate_icon` [method](https://github.com/Automattic/jetpack/blob/master/projects/packages/jitm/src/class-jitm.php#L199) currently supports the 'jetpack' and 'woocommerce' icons. Any other string will result in no icon being displayed.

I think that we want to allow pre-connection JITMs to be displayed without an icon. What do you think of using an empty string to indicate that no icon should be displayed? Then, we could check that the value is one of the accepted values or an empty string.

Reply:
Yeah, I think that would be fine 👍